### ISSUE TITLE
Issue with AirVPN (EddieUI) : Can't secure Ipv6 connection

### ISSUE THREAD

**Issue by Klemet** [henrypp/simplewall#1112]
Hello there !

Thank you so much for the wonderful piece of software that is Simplewall. I'm having a great time with it.

I'm just a bit stuck concerning its compatibility with the VPN I'm using, AirVPN, along with its desktop client ([EddieUI](https://github.com/AirVPN/Eddie)). 

Whatever I do, EddieUI gets stuck when trying to secure an Ipv6 connection. I've been tweaking Simplewall again and again, and tried to watch for connections that might give me a clue as to what service, program or rule I should authorize. But to no avail ! 

![image](https://user-images.githubusercontent.com/41070328/144612267-3eb0e854-16c4-4aa2-b020-07e9e90af7aa.png)

The only way for EddieUI to connect is to disable simplewall's filters. The weirdest thing is that once that the VPN is connected, simplewall can be reactivated, and everything work well. So, the problem seems to only be during the initialization of the VPN connection.

Would anybody have a clue as to what to do to solve this ?

**Comment by Pandaa-P** [comment]
>I've been tweaking Simplewall again and again, and tried to watch for connections that might give me a clue as to what service, program or rule I should authorize. But to no avail !

If no message appears you can add the app manually to simplewall from `File` then `Add App` and then allow your vpn app to access the internet.

**Comment by Klemet** [comment]
> If no message appears you can add the app manually to simplewall from `File` then `Add App` and then allow your vpn app to access the internet.

Sorry for not writing this detail; the app in itself is already in the list, and is able to access the internet. That's what's pretty weird.

As I wrote, it's just this "Checking route IPv6" that has a problem with simplewall; if simplewall's filters are disabled for the time of this step, then the VPN connects without issues. Once the VPN is past this step, reactivating simplewall's filters doesn't not cut the connection, and the VPN will stay connected.

In short, it's already in the list, and it can connect to the internet; it's just this very step that has an issue with simplewall's filters. And I can't find the tweak to fix it.

**Comment by jacob755** [comment]
if logging enabled in SW there should definitely log something if its blocked
if you have disabled any rule in system rules check them
check if ICMPv6 in user rules ticked
allow loopback allow ip6 redirect untick block inbound ,block outbound
does airvpn connect if you tick for ip4 only connection?

**Comment by Klemet** [comment]
Thanks for the recommendations, @jacob755 !

- Despite the log enabled, I was unable to find with precision what was getting blocked.
- I've activated every system rule, ticked ICMPv6, allowed loopback, ip6, etc.
- If I forced AirVPN to use only an ip4 connection, the connection was made without any issues despite the filters of SW being in place.

Just after that, I did manage to make it work with IPv6, though ! It seems like the answer might have been in the ICMPv6 rule. By curiosity, I edited the rule to look at it, and found it that in the "Application" tab of the rule, `eddie.exe` (the GUI of AirVPN) was ticked; which meant that no other un-ticked application in the rule was autorized (since you must leave ALL apps un-ticked in order for the rule to apply to all of them).

Once I un-ticked `eddie.exe` from the application list of the rule, and applied it, everything worked. I really don't know why `eddie.exe` came to be ticked in the rule; but my guess is that the connection depended on another application than was not included in the rule anymore.

I'd say that the issue is now solved. For future users : remember to check the content of user or system rules to see if one of them is not applied only to a single app.

Thank you for your help !
